### PR TITLE
chore: open read me happy path

### PR DESCRIPTION
**Pull request by 1openwindow**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_sprints/taskboard/Teams%20Extensibility%20E2E%20Team/Microsoft%20Teams%20Extensibility/CY22-2.1?workitem=13133655

### CODE REVIEW

File: packages/vscode-extension/src/handlers.ts [hunk 1]
```diff
@@ -1155,6 +1155,39 @@ async function openMarkdownHandler() {
   }
 }
 
+export async function openReadMeHandler(args: any[]) {
+  if (workspace.workspaceFolders && workspace.workspaceFolders.length > 0) {
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/handlers.ts:1159]: What if current project is not a valid teamsfx project?

File: packages/vscode-extension/src/handlers.ts [hunk 2]
```diff
@@ -1155,6 +1155,39 @@ async function openMarkdownHandler() {
...
+        targetFolder = botFolder;
+      }
+    }
+    // When tab and bot coexist, readme file would reside in project root folder.
+    // Naming it README.md could accidently overwrite our users' own readme file.
+    // So we name it README-auto-generated.md here.
+    const autoGeneratedReadmePath = `${targetFolder}/${AutoGeneratedReadme}`;
+    const uri = (await fs.pathExists(autoGeneratedReadmePath))
+      ? Uri.file(autoGeneratedReadmePath)
+      : Uri.file(`${targetFolder}/README.md`);
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/handlers.ts:1182]: What if both path not exist?

File: packages/vscode-extension/src/handlers.ts (hunk 1 above)
Comment (1openwindow) [reply packages/vscode-extension/src/handlers.ts:1159]: this PR is only for happy path. For this case, it is still under discussion. there should be other PR to take care this case.

**Review by HuihuiWu-Microsoft — APPROVED** [review]
Don't forget to update logic when discussion is done

**Pull request merged**